src/nft/utils/listNfts.ts

```diff
@@ -59,7 +59,7 @@ const getConsiderationItems = (
   openSeaFee?: ConsiderationInputItem
 } => {
   const creatorFeeBasisPoints = asset?.basisPoints ?? 0
-  const openSeaBasisPoints = !asset?.basisPoints ? 50 : 0
+  const openSeaBasisPoints = (ListingMarkets.find((market) => market.name === 'OpenSea')?.fee ?? 0) * 100
```

Comment:
can this not just be `ListingMarkets[1].fee` then you won't need the `?`s

Reply:
this allows us to be able to modify the array without causing downstream effects